code/modules/admin/verbs/randomverbs.dm

```diff
@@ -781,7 +786,9 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(shuttle_master.emergency.mode >= SHUTTLE_DOCKED)
 		return
 
+	shuttle_master.emergency.canRecall = TRUE // Resets incase the admin recalls a round ender for some reason.

```

Comment:
This might lead to admins unintentionally breaking things?

Reply:
nothing in the gamemodes that call shuttles would break anything since they have standard round end conditions, but I can definitely see the concern. My main issue is that currently if a mistake is made and a shuttle is called, they would have to jump to the emergency shuttle dock, var edit it, then recall if they wanted to. Think a prompt would suffice or just remove it entirely?

Reply:
A prompt with "Yes" automatically selected (yes being "make shuttle recallable") would probably be sufficient, plus a warning of "Don't click anything else if you don't know what this does".

Reply:
in an alert, the first option is the autoselected one, yes? if not then I might need a quick explanation of the gui you're requesting

Reply:
I believe it would just be
```dm
if(alert("Set Shuttle Recallable (Select Yes unless you know what this does)", "Recallable?", "Yes", "No") == "Yes")
    shuttle_master.emergency.canRecall = TRUE
```

Reply:
I'll toss that in quick along with the other suggestions.